.github/sync_upstream.sh

```diff
@@ -23,6 +23,9 @@ git fetch "imports/openjfx/jfx-dev/rt/root"
 echo "Merge (root)"
 git merge "imports/openjfx/jfx-dev/rt/root/master" -m "Merge from (root)" --no-ff
 
+echo "Pull master (github)"
+git pull github master
```

Comment:
Should this be done before the previous git merge? That matches what is done for develop (it probably doesn't matter, but might be cleaner).

Reply:
The order does indeed not matter, but I've moved it after the checkout making the steps analogous to how develop branch is handled.